src/reducers/pois.reducer.js

```diff
@@ -40,10 +43,16 @@ export default function pois(state = initialState.pois, action) {
           poi => poi.id !== action.payload.id
         )
       }
+    case POI_FORM_MEDIA_REMOVED:
+      return {
+        ...state,
+        carouselIndex: (state.carouselIndex = 0) ? state.carouselIndex - 1 : 0

```

Comment:
this is assigning `state.carouselIndex` to `0` (im guessing by accident). use a `===`